change/@fluentui-react-components-345f2847-24c1-45d8-a4d2-e6d692958a50.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Moving slider version to stable.",
```

Comment:
I struggled between the two, it would normally be of type `chore` but it is the incorrect version now so it is also technically a `fix`. I'm fine with either classification.

Reply:
I see, then both could work. You can ignore these comments :)